frontend/src/components/NewRunParametersV2.tsx

```diff
@@ -130,6 +130,7 @@ function generateInputValidationErrMsg(
 }
 
 function NewRunParametersV2(props: NewRunParametersProps) {
+  const { specParameters, setIsValidInput } = props;
```

Comment:
I don't fully get the change here. Can you describe a little more details? Thank you.